### PR TITLE
Regression: Attachment auto size

### PR DESCRIPTION
**Pull request by hugocostadev**
## Proposed changes (including videos or screenshots)

 Attachment Image or Image Preview size in small screens are not respecting container boundaries
 
 **Before:**
image

**NOW:**
image

## Further comments

TC-530

### CODE REVIEW

**Review by ggazzo — CHANGES_REQUESTED** [review]
is this not enough ?
```css
.gallery-item {
    max-width: 100%;

}
```

File: apps/meteor/client/components/message/content/urlPreviews/UrlImagePreview.tsx [hunk 1]
```diff
@@ -6,10 +6,10 @@ import { useOembedLayout } from '../../hooks/useOembedLayout';
 import type { UrlPreviewMetadata } from './UrlPreviewMetadata';
 
 const UrlImagePreview = ({ url }: Pick<UrlPreviewMetadata, 'url'>): ReactElement => {
-	const { maxWidth: oembedMaxWidth, maxHeight: oembedMaxHeight } = useOembedLayout();
+	const { maxHeight: oembedMaxHeight } = useOembedLayout();
 
 	return (
-		<Box maxHeight={oembedMaxHeight} maxWidth={oembedMaxWidth}>
+		<Box maxHeight={oembedMaxHeight} maxWidth={'100%'}>
```
Comment (tassoevan) [inline apps/meteor/client/components/message/content/urlPreviews/UrlImagePreview.tsx:12]: Why not set `width='100%'` and keep `maxWidth={oembedMaxWidth}`?

**Review by tassoevan — DISMISSED** [review]

**Review by ggazzo — DISMISSED** [review]

**Review by KevLehman — COMMENTED** [review]
Ia the .eslintcache file needed?

**Comment by ggazzo** [comment]
> Ia the .eslintcache file needed?

if we want to speedup CI task and use the lint. yes it is

and it mean @hugocostadev has run at least once the eslint on his machine

**Review by ggazzo — APPROVED** [review]

**Pull request merged**